### PR TITLE
Option to provide plugins for babylon parser

### PR DESCRIPTION
**Pull request by yuri1992**

### CODE REVIEW

**Comment by dsingleton** [comment]
Hi @fraserxu 👋 

I'm an engineer at @deliveroo, where we're using this module. First off, thanks for the work you've been doing maintaining it.

We're having issues with some bleeding edge JS features causing parse errors. This PR would help us solve it by specifying plugins via the CLI, without having to make changes to this repo directly (thanks @yuri1992!).

Do you think this PR is something you'd be happy to merge in, and cut a new release for? 

This PR feels 95% there. There's a small lint error - and maybe `jsx` should always be included in the plugin list (if you specify plugins plugins you need to re-include it).

If it needs additional work we'd be happy to help. We'd much rather contribute back than create another fork :)

**Comment by dsingleton** [comment]
If it helps, i've [made a branch](https://github.com/fraserxu/babel-jsxgettext/compare/master...deliveroo:plugin-cli?expand=1) that includes the above linting and default-plugin changes. If it's easier I can open that as a fresh PR. 👍

File: bin/cli.js [hunk 1]
```diff
@@ -23,11 +24,12 @@ var cli = meow({
   ].join('\n')
 })
 
+var plugins = cli.flags.plugins ? cli.flags.plugins.split(",") : null
```
Comment (fraserxu) [inline bin/cli.js:27]: Would you mind change `"` to `'` so the standard lint check could pass?

**Comment by fraserxu** [comment]
Thanks @dsingleton , would you mind create a new pr for that?

**Pull request merged**